packages/react/src/shared/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from '../internals/getQueryKey';
+
+export {
+  createReactQueryUtilsProxy,
+  type DecoratedProcedureUtilsRecord,
+} from '../internals/utilsProxy';
```

Comment:
should we move these stuff out somewhere or just mark them `@internal` now that they can be imported elsewhere?

Reply:
Yeah, that'd be nice 👌

Reply:
You can just add a `FIXME` comment too :D